spec/graphql/mutations/delete_account_mutation_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'deleteAccount mutation' do
+  let!(:user) { create :user, password: 'changemenow' }
```

Comment:
Please use
```ruby
let!(:user) { create :user }
let(:password) { user.password }
```
and use the password variable everywhere.

The same applies to the other specs.